backend/copr_backend/storage.py

```diff
@@ -206,6 +267,60 @@ def delete_repository(self, chroot):
         self.client.delete_repository(repository)
         self.client.delete_distribution(distribution)
 
+    def delete_project(self, dirname):
+        prefix = "{0}/{1}".format(self.owner, dirname)
+        response = self.client.list_repositories(prefix)
+        repositories = response.json()["results"]
+        for repository in repositories:
+            self.client.delete_repository(repository["pulp_href"])
```

Comment:
this unfortunately doesn't work :/ After I deleted that project, the directory structure remained inside pulp instance - e.g. I still have `/nikromen/pulp-test/fedora*`. But then `ls` inside the directory complains:
```
ls: Access failed: 404 Distribution is not pointing to a publication, repository, repository version, or remote. (/pulp/content/nikromen/pulp-test/fedora-40-x86_64/)
```

Reply:
Good catch, thank you. Turns out I needed to remove the distribution as well. Fixed.